yjit/src/disasm.rs

```diff
@@ -192,6 +187,44 @@ pub fn disasm_addr_range(cb: &CodeBlock, start_addr: usize, end_addr: usize) ->
     return out;
 }
 
+#[cfg(not(feature = "disasm"))]
```

Comment:
Could use a short comment here to explain that this version of the function is used when not built with the disasm feature, will only dump bytes.